checker/tests/src/tests/OIGJTest.java

```diff
@@ -21,8 +22,8 @@ public OIGJTest(File testFile) {
     }
 
     @Parameters
-    public static Collection<Object[]> data() {
-        return filter(testFiles("oigj", "all-systems"));
+    public static Collection<Object[]> getTestFiles() {
+        return filter(TestUtilities.findNestedJavaTestFiles("oigj", "all-systems"));
     }
 
     // Duplicate from JavariTest.
```

Comment:
I agree.  Perhaps we can put an issue to fix both the IGJ/OIGJ tests together and somebody can handle that when they get a chance.  However, I worry that it's low priority so it might never get done.  Then again, at least this would be an easy task for a new hire, one in which they could make forward progress right away.